lib/models/Secret.js

```diff
@@ -0,0 +1,46 @@
+const pool = require('../utils/pool');
+
+class Secret {
+  id;
+  title;
+  description;
+  createdAt;
+
+  constructor(row) {
+    this.id = row.id;
+    this.title = row.title;
+    this.description = row.description;
+    this.createdAt = row.createdAt;
+  }
+
+  static async getAll() {
+    const { rows } = await pool.query(
+      ` SELECT * 
+    FROM secrets`
+    );
+    return rows.map((row) => new Secret(row));
+  }
+  static async insert({ title, description }) {
+    const { rows } = await pool.query(
+      `INSERT into secrets (title, description) 
+      VALUES ($1, $2)
+      RETURNING *`,
+      [title, description]
+    );
+    return new Secret(rows[0]);
+  }
+}
+module.exports = { Secret };
+
+// static async insert({ title, description }) {
```

Comment:
if you have to comment out 11 lines of code that are the same as the preceding 11 lines, you should just delete it.